scripts/aws-deploy-local-layer.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
I think we might want to convert this script into a node script at some point in the future because we recently moved from bash scripts to node scripts for platform independence (see https://github.com/getsentry/sentry-javascript/issues/4720 for context). But this can easily be done some time in the future in a follow-up PR since it's a new script and our contributors won't need it

Reply:
Sorry - this didn't get rebased in time. That's just a local script Anton made, which is gone in the end result.

Reply:
Ahh that makes sense 👍